spark-indexer/src/main/java/indexer/ValidateSavedDocument.java

```diff
@@ -0,0 +1,49 @@
+package indexer;
+
+import java.util.ResourceBundle;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.solr.common.SolrInputDocument;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import indexer.util.SparkUtils;
+
+/**
+ * @author lgonzales
+ * @since 2019-12-12
+ */
+@Slf4j
+public class ValidateSavedDocument {
+
+    public static void main(String[] args) {
+        ResourceBundle applicationConfig = SparkUtils.loadApplicationProperty();
+        JavaSparkContext sparkContext = SparkUtils.loadSparkContext(applicationConfig);
+
+        String hdfsFilePath =
+                applicationConfig.getString(
+                        "uniprot.solr.documents.path"); // applicationConfig.getString(args[0]);
+        JavaRDD<SolrInputDocument> solrInputDocumentRDD =
+                (JavaRDD<SolrInputDocument>)
+                        sparkContext
+                                .objectFile(hdfsFilePath)
+                                .map(
+                                        obj -> {
+                                            return (SolrInputDocument) obj;
+                                        });
+
+        solrInputDocumentRDD
+                .take(200)
```

Comment:
This class was used for a test, to validate if we were saving solr document properly in hdfs, so it just print first 200 documents and print it. I think it can be manually changed if we need it, because it is a class to test/help during the development phase.  I just documented it.. is it okay?

Reply:
Yes if it's documented then it's fine